tests/common/validation.rs

```diff
@@ -0,0 +1,81 @@
+// SPDX-FileCopyrightText: 2022 Profian Inc. <opensource@profian.com>
+// SPDX-License-Identifier: AGPL-3.0-only
+
+#![allow(dead_code)]
```

Comment:
Should not need this, just remove the dead code

Reply:
File deleted.